nvidia/bmi160_spi_demo/index.rst

```diff
@@ -0,0 +1,15 @@
+.. _bmi160_spi_demo:
+
+BMI160 SPI Demo
+###############
+
+| The chapter shows **How to Test SPI** and **How to Make SPI Demo** with **BMI160**.
```

Comment:
You can mention the supported model of BMI160 here.
Supported model: ROScube Pico series

Reply:
Done.